sql/views.sql

```diff
@@ -0,0 +1,28 @@
+-- Copyright (c) 2016-2018  Timescale, Inc. All Rights Reserved.
+--
+-- This file is licensed under the Apache License, see LICENSE-APACHE
+-- at the top level directory of the TimescaleDB distribution.
+
+
+-- Convenience view to list all hypertables and their space usage
+CREATE OR REPLACE VIEW information_schema._timescaledb_hypertables AS
```

Comment:
I know there's been quite a bit of discussion around which schema to put this view in, but I haven't seen a strong argument against creating a Timescale view schema, and sticking our Timescale-specific views in this special schema. I think this schema is acceptable for several reasons: 1) we create schemas for our other metadata tables, so we have a habit of creating schemas anyway, 2) I have a number of other views in mind, and I can even imagine that we actually create one view for each of the internal metadata tables that we have.